ArchiSteamFarm/Bot.cs

```diff
@@ -534,8 +538,17 @@ public sealed class Bot : IAsyncDisposable {
 					continue;
 				}
 
+				byte maxSetsAllowed = (byte) (maxItems - result.Count);
```

Comment:
Since `maxItems` is `ushort` now, we have to correct it into `ushort` as well. Even if we make all those operations, we can't be sure it won't be higher than `255`.

Reply:
Sorry about that. I totally forgot about the other ones